raiden_libs/test/mocks/client.py

```diff
@@ -241,6 +241,22 @@ def get_monitor_request(
         )
         return monitor_request
 
+    @assert_channel_existence
+    def get_fee_info(self, partner_address, **kwargs) -> FeeInfo:
+        """Get a signed fee info message for an open channel.
+        Parameters:
+            partner_address - address of a partner the node has channel open with
+            **kwargs - arguments to FeeInfo constructor
+        """
+        channel_id = self.partner_to_channel_id[partner_address]
+        fi = FeeInfo(
```

Comment:
Just a note for the future: Please don't use such abbreviations for variable names. Use more descriptive names that are easiers to understand. e.g. `fee_info`.